System/Process/Internals.hs

```diff
@@ -38,6 +38,7 @@ module System.Process.Internals (
     withFilePathException, withCEnvironment,
     translate,
     createPipe,
+    createPipeFD,
```

Comment:
It seems common to abbreviate "file descriptor" as `Fd`. Given that this convention is used below perhaps it would make sense to preserve consistency here?